zmdocument.cls

```diff
@@ -241,6 +246,7 @@
 
 %%% LINKS
 \newcommand{\externalLink}[2]{\href{#2}{\color{externalLinkColor}{#1~\linkIcon}}}
+\newcommand{\externalCleanLink}[2]{\href{#2}{\color{white}{#1}}}
```

Comment:
Why ... "clean" ?

Reply:
(you can answer that one in french :p )

Reply:
Clean because it's doesn't add icon for external link.

Reply:
Mmmh ... That's not a very explicit name for that. Especially since the link is rendered in white, which indicate that it should be used were the background is not (please use a named color). `\specialLink` could do it, although not very explicit either, `\titlePageLink`, maybe ?

Also, I would rather prefer that this command remains internal to the template (by prefixing it with an `@`).